GCC 12.3 warning while building USB Video

### Operating System

Linux

### Arduino IDE version

IDE 1.x

### Board

RPi Pico

### ArduinoCore version

arduino-pico master

### TinyUSB Library version

3.0.0

### Sketch as ATTACHED TXT

Any build

### Compiled Log as ATTACHED TXT

````
/home/earle/Arduino/hardware/pico/rp2040/libraries/Adafruit_TinyUSB_Arduino/src/arduino/video/Adafruit_USBD_Video.cpp: In member function 'virtual uint16_t Adafruit_USBD_Video::getInterfaceDescriptor(uint8_t, uint8_t*, uint16_t)':
/home/earle/Arduino/hardware/pico/rp2040/libraries/Adafruit_TinyUSB_Arduino/src/arduino/video/Adafruit_USBD_Video.cpp:235:29: warning: missing initializer for member 'tusb_desc_endpoint_t::<unnamed struct>::usage' [-Wmissing-field-initializers]
  235 |              .bInterval = 1}};
      |                             ^
````

### What happened ?

GCC 12.3 warning as shown in the log

### How to reproduce ?

Build any sketch with `-Wall -Wextra` to see the warning

### Debug Log

_No response_

### Screenshots

_No response_

--- a/src/arduino/video/Adafruit_USBD_Video.cpp
+++ b/src/arduino/video/Adafruit_USBD_Video.cpp
@@ -230,7 +230,7 @@ uint16_t Adafruit_USBD_Video::getInterfaceDescriptor(uint8_t itfnum_deprecated,
              .bDescriptorType = TUSB_DESC_ENDPOINT,
 
              .bEndpointAddress = ep_in,
-             .bmAttributes = {.xfer = TUSB_XFER_BULK, .sync = 0},
+             .bmAttributes = {.xfer = TUSB_XFER_BULK, .sync = 0, .usage = 0},
              .wMaxPacketSize = 64,
              .bInterval = 1}};
 

